src/data/markdown/translated-guides/en/03 Results output/200 Real-time/00 Prometheus remote write.md

```diff
@@ -31,108 +32,153 @@ All k6 metric types are converted into an equivalent Prometheus remote write typ
 | Counter | Counter                   | `k6_*_total`         |
 | Gauge   | Gauge                     | `k6_*_<unit-suffix>` |
 | Rate    | Gauge                     | `k6_*_rate`          |
-| Trend   | Gauges / Native Histogram | `k6_*_<unit-suffix>` |
+| Trend   | [Gauges (default)](#trend-stats-option) or [Native Histogram](#prometheus-native-histogram) | `k6_*_<unit-suffix>` |
+
+The output maps the metrics into time series with Name labels.
+As much as possible, k6 respects the [naming best practices](https://prometheus.io/docs/practices/naming) that the Prometheus project defines:
+
+* All time series are prefixed with the `k6_` namespace.
+* All time series are suffixed with the base unit of the sample value (if k6 knows what the base unit is).
+* Trends and rates have the relative suffixes, to make them more discoverable.
+
+## Trend metric conversions
+
+This output provides two distinct mechanisms to send [built-in and custom Trend metrics](https://k6.io/docs/using-k6/metrics/) to Prometheus:
```

Comment:
@codebien double checking, does this extension send also custom metrics?

The last version said: `all the generated data points for the built-in k6 metrics`

Reply:
```suggestion
This output provides two distinct mechanisms to send [built-in and custom metrics](https://k6.io/docs/using-k6/metrics/) to Prometheus:
```
It flushes **all** (built-in and customs) tracked metrics. In the future, if we will have an API for defining the _tracked metrics_ then it will flush only enabled metrics.